Infrastructure/Factories/QualityAssuredSpreadsheetTuitionPartnerFactory.cs

```diff
@@ -3,85 +3,89 @@
+    private static readonly IDictionary<(Domain.Enums.TuitionType, int), (string, int)> SubjectPricesCellReferences = new Dictionary<(Domain.Enums.TuitionType, int), (string, int)>
         {
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage1English), ("C", 7) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage1Maths), ("D", 7) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage1Science), ("E", 7) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage2English), ("C", 17) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage2Maths), ("D", 17) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage2Science), ("E", 17) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage3English), ("C", 27) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage3Humanities), ("D", 27) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage3Maths), ("E", 27) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage3ModernForeignLanguages), ("F", 27) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage3Science), ("G", 27) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage4English), ("C", 37) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage4Humanities), ("D", 37) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage4Maths), ("E", 37) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage4ModernForeignLanguages), ("F", 37) },
-            { (TuitionTypes.InSchool, Subjects.Id.KeyStage4Science), ("G", 37) },
-
-            { (TuitionTypes.Online, Subjects.Id.KeyStage1English), ("C", 47) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage1Maths), ("D", 47) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage1Science), ("E", 47) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage2English), ("C", 57) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage2Maths), ("D", 57) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage2Science), ("E", 57) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage3English), ("C", 67) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage3Humanities), ("D", 67) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage3Maths), ("E", 67) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage3ModernForeignLanguages), ("F", 67) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage3Science), ("G", 67) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage4English), ("C", 77) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage4Humanities), ("D", 77) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage4Maths), ("E", 77) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage4ModernForeignLanguages), ("F", 77) },
-            { (TuitionTypes.Online, Subjects.Id.KeyStage4Science), ("G", 77) }
+            { (Domain.Enums.TuitionType.InSchool, Subjects.Id.KeyStage1English), ("C", 7) },
```

Comment:
I suggest we declare a namespace `using TuitionType = Domain.Enums.TuitionType;` at the top and use `TuitionType` instead of `Domain.Enums.TuitionType`  where we refer to the `TuitionType` enum.

Reply:
Agreed, change made.